packages/atomic/src/components/search/atomic-search-interface/atomic-search-interface.ts

```diff
@@ -510,17 +590,11 @@ export class AtomicSearchInterface
         this.searchStatus.state.firstSearchExecuted &&
         !this.searchStatus.state.hasError;
 
-      this.host.classList.toggle(
-        noResultsSelector,
-        hasNoResultsAfterInitialSearch
-      );
+      this.classList.toggle(noResultsSelector, hasNoResultsAfterInitialSearch);
```

Comment:
I wonder if we could take an approach similar to what we did in the atomic-modal (i.e., relying on attributes for styling instead of toggling classes)...

but it seems to be ipx tthat's using these classes. Not sure if we can actually do something without a breaking change.